packages/forms/src/elements/address/addressLookup.tsx

```diff
@@ -145,8 +146,15 @@ export const AddressLookup: React.FC<AddressProps> = ({
 					selectAddressRequiredMessage={selectAddressRequiredMessage}
 					noAddressesFoundMessage={noAddressesFoundMessage}
 					onValidatePostcode={(isValid) =>
-						onValidatePostcode ? onValidatePostcode(isValid) : null
+					{
+						if(onAddressChanging && onAddressChanging !== null && isValid){
```

Comment:
Checking for `onAddressChanging !== null` won't do anything here because if it is `null` it will already have failed the first test, which is checking whether `onAddressChanging` is any value that is 'falsy', which includes checking for `null`.

The same applies on line 167.